.gitignore

```diff
@@ -41,3 +41,6 @@ private-key.pem
 #nyc test coverage
 .nyc_output
 coverage
+
+# prevent the assets folder to be published
+assets
```

Comment:
This will ignore the assets folder altogether, not only for publish to npm. You either have to put it to `.npmignore` or you define a specific list of files / folders that you want to be published as described at https://docs.npmjs.com/files/package.json#files, which is what I’d prefer. Here is an example: https://github.com/octokit/rest.js/blob/master/package.json#L110-L115

Reply:
Ah, I misunderstood the issue, made the changes, please see if this is appropriate. :+1: 

Thanks for the example though. :)